src/scenes/Dashboard/ProgressReportsWidget/ProgressReportsGrid.tsx

```diff
@@ -98,6 +99,19 @@ export const ProgressReportsColumnMap = {
     filterable: false,
     valueGetter: (_, row) => row.varianceExplanation.scheduleStatus,
   },
+  cumulativeSummary: {
+    headerName: 'Cumulative Progress',
+    minWidth: 250,
```

Comment:
I think a slightly smaller minWidth tightens this up, 210 seemed to be a decent size.